setup.py

```diff
@@ -62,7 +62,7 @@ def get_cmake_args():
           "test": ["pytest", "matplotlib", "xarray", "pandas", "pythreejs"],
           'all': ['h5py', 'scipy>=1.7.0', 'graphviz', 'pooch'],
           'interactive': [
-              'ipympl', 'ipython', 'ipywidgets', 'matplotlib', 'jupyterlab',
+              'ipympl', 'ipython', 'ipywidgets', 'matplotlib>=3.5', 'jupyterlab',
```

Comment:
I think you are missing a few places:
- `requirements/extra.in`
- `docs/environments/scipp.yml` (`matplotlib-base`)
- `docs/environments/developer.yml` (`matplotlib-base`)

Reply:
Done, thanks!